src/ACEtk/interpretation/Region/src/ctor.hpp

```diff
@@ -0,0 +1,14 @@
+Region( int INT, xRange&& x, yRange&& y )
+try:
+  interpolationScheme_( verifyInterpolationScheme( INT ) ),
+  x_( x ), y_( y )
```

Comment:
You're taking rvalue references to x and y. Don't copy. Move instead.

Reply:
Yeah, this didn't feel right, but I wasn't quite sure how to do this.